### ISSUE TITLE
noscriptをつける

### ISSUE THREAD

**Issue by naoki-umeyama** [tokyo-metropolitan-gov/covid19#846]
## 改善詳細 / Improve details
- noscriptがない
- Lighthouseで、Does not provide fallback content when JavaScript is not available の警告が出てている
- JavaScript Off の状態でアクセスすると、ローディングのUIが表示され続けており、ユーザに次のアクションを期待させてしまう。
- [当サイトについて](https://stopcovid19.metro.tokyo.lg.jp/about)内でJavaScriptに関しての記述があるが、JavaScript Off のユーザは該当画面を確認するすべがない

## スクリーンショット / Screenshot
- Lighthouse警告
![76143991-30162880-60bf-11ea-9d29-a09215502184](https://user-images.githubusercontent.com/27715262/76165575-7a6cd780-619b-11ea-9649-eeac2c1db971.png)

- JavaScript Off 時の画面
![76143846-16c0ac80-60be-11ea-8324-1e7698eea95d](https://user-images.githubusercontent.com/27715262/76165603-bacc5580-619b-11ea-9f3a-a95399208fd5.png)

## 期待する見せ方・挙動 / Expected behavior
- noscriptを追加する
- JavaScript Off でアクセスした際に「JavaScriptを有効にして頂きますようお願いいたします」の文言を表示すること

## 動作環境・ブラウザ / Environment
- macOS / Windows / Linux / iOS / Android
- Chrome / Safari / Firefox / Edge / Internet Explorer

**Comment by halsk** [comment]
すみません。一度 #759 を取り込んだものの、

```
DOMException: Failed to execute 'appendChild' on 'Node': This node type does not support this method.
```
が出てエラーとなったため、Revert しました。
修正して、再度 PR いただけないでしょうか。

**Comment by smellman** [comment]
https://github.com/nuxt/nuxt.js/issues/2582
`<noscript inline-template>` とinline-templateが必用だと思われる。

**Comment by naoki-umeyama** [comment]
失礼しました。 上記修正対応ありがとうございます。
